packages/web/package.json

```diff
@@ -63,6 +63,7 @@
     "@types/react-dom": "^18.3.0",
     "@vitejs/plugin-react-swc": "^3.7.0",
     "autoprefixer": "^10.4.20",
+    "chokidar": "^3.6.0",
```

Comment:
They just released 4.0.0 of the lib and it looks to have substantial improvements. Can we use that version?

Reply:
With @benjreinhart on this, bumping to 4.0 gets rid of a bunch of node deps and makes things simpler, but their changelog does state glob support was removed so I'm not sure what would be needed to keep the `../api/**/*.mts` behaviour

Reply:
I’ve upgraded chokidar to v4 and adapted the codebase @benjreinhart @versecafe